core/src/main/java/com/klaytn/caver/transaction/type/FeeDelegatedSmartContractExecution.java

```diff
@@ -0,0 +1,365 @@
+
+        List<RlpType> feePayerSignatures = ((RlpList) (values.get(9))).getValues();
+        List<SignatureData> feePayerSignList = SignatureData.decodeSignatures(feePayerSignatures);
+
+        FeeDelegatedSmartContractExecution feeDelegatedSmartContractExecution = new FeeDelegatedSmartContractExecution.Builder()
+                .setNonce(nonce)
+                .setGas(gas)
+                .setGasPrice(gasPrice)
+                .setTo(to)
+                .setValue(value)
+                .setFrom(from)
+                .setInput(input)
+                .setSignatures(senderSignList)
+                .setFeePayer(feePayer)
+                .setFeePayerSignatures(feePayerSignList)
+                .build();
+
+        return feeDelegatedSmartContractExecution;
+    }
+
+    /**
+     * Returns the RLP-encoded string of this transaction (i.e., rawTransaction).
+     * @return String
+     */
+    @Override
+    public String getRLPEncoding() {
+        // type + encode([nonce, gasPrice, gas, to, value, from, input, txSignatures, feePayer, feePayerSignatures])
+        this.validateOptionalValues(false);
+
+        List<RlpType> senderSignatureRLPList = new ArrayList<>();
+        List<RlpType> feePayerSignatureRLPList = new ArrayList<>();
+
+        for(SignatureData signatureData : this.getSignatures()) {
+            senderSignatureRLPList.add(signatureData.toRlpList());
+        }
+
+        for(SignatureData signatureData : this.getFeePayerSignatures()) {
+            feePayerSignatureRLPList.add(signatureData.toRlpList());
+        }
+
+        List<RlpType> rlpTypeList = new ArrayList<>();
+        rlpTypeList.add(RlpString.create(Numeric.toBigInt(this.getNonce())));
+        rlpTypeList.add(RlpString.create(Numeric.toBigInt(this.getGasPrice())));
+        rlpTypeList.add(RlpString.create(Numeric.toBigInt(this.getGas())));
+        rlpTypeList.add(RlpString.create(Numeric.hexStringToByteArray(this.getTo())));
+        rlpTypeList.add(RlpString.create(Numeric.toBigInt(this.getValue())));
+        rlpTypeList.add(RlpString.create(Numeric.hexStringToByteArray(this.getFrom())));
+        rlpTypeList.add(RlpString.create(Numeric.hexStringToByteArray(this.getInput())));
+        rlpTypeList.add(new RlpList(senderSignatureRLPList));
+        rlpTypeList.add(RlpString.create(Numeric.hexStringToByteArray(this.getFeePayer())));
+        rlpTypeList.add(new RlpList(feePayerSignatureRLPList));
+
+        byte[] encodedTransaction = RlpEncoder.encode(new RlpList(rlpTypeList));
+        byte[] type = new byte[] { (byte)TransactionType.TxTypeFeeDelegatedSmartContractExecution.getType() };
+        byte[] rawTx = BytesUtils.concat(type, encodedTransaction);
+
+        return Numeric.toHexString(rawTx);
+    }
+
+    /**
+     * Returns the RLP-encoded string to make the signature of this transaction.
+     * @return String
+     */
+    @Override
+    public String getCommonRLPEncodingForSignature() {
```

Comment:
Don't do "this.validateOptionalValues(false)" here?

Reply:
I missed this function. I'll modify it.

Reply:
I missed this function. I modified it